### ISSUE TITLE
runtime CUDA error for custom data

### ISSUE THREAD

**Issue by 2dongheee** [zylo117/Yet-Another-EfficientDet-Pytorch#570]
I attempt to proceed with learning on custom data.
However, learning does not proceed with errors.
Error does not appear only when obj_list is "80 class". What's the cause?
I think the size is different from the input output. and the tensor dimension from loss? Size? problem, but I don't know

![image](https://user-images.githubusercontent.com/48238661/102857854-8bf24800-446c-11eb-9e1f-6e5dfff9ffe3.png)

![image](https://user-images.githubusercontent.com/48238661/102857771-5d746d00-446c-11eb-9580-d8c4a0ca6505.png)

**Comment by zylo117** [comment]
No, the number of coco classes in the repo is 90. There are empty strings '' in the list.
And you should check your category ids of your dataset, which should start from 1.

**Comment by 2dongheee** [comment]
Thank you for your answer. :)
But I have newly constructed Dataloader instead of pycocotools, due to data format related issues.
As a result, only the image and annotation file are replaced and the rest of the code remains the same.

**Comment by nTnZone** [comment]
> file are replaced and the rest of the code remains the sam

hello @2dongheee , I have same question with you in cuda error, could tell me how to fix it? I  really want to fix it, can you cantact with me on email plz 0v0. 995649319@qq.com
